api/renderers.py

```diff
@@ -6,16 +6,16 @@
 
 from django.db.models import Manager, QuerySet
 from django.utils import encoding, six
-from rest_framework import relations
+from rest_framework import relations, renderers
 from rest_framework.serializers import (BaseSerializer, ListSerializer,
                                         Serializer)
 from rest_framework.settings import api_settings
 
 import inflection
-from rest_framework_json_api import renderers, utils
+from rest_framework_json_api import utils
 
 
-class JSONRenderer(renderers.JSONRenderer):
+class JSONAPIRenderer(renderers.JSONRenderer):
```

Comment:
around [L164](https://github.com/CenterForOpenScience/lookit-api/blob/e8eb8e7093d94b33c5a341eccd290beddc0dfdb2/api/renderers.py#L162) 
```
relation_data = {
       'data': resource.get(field_name)
}
```
- Do we need to be including the id/type of every related resource?  These items aren't paginated, so if you have 200 responses to a study, the type/id of 200 responses will be included in the study > responses relationship. Relationships aren't required to have a "data" key.
<img width="361" alt="screen shot 2017-06-29 at 3 16 53 pm" src="https://user-images.githubusercontent.com/9755598/27706059-0e74ce62-5cde-11e7-9e0e-95b3fcf57d29.png">